src/main/java/com/google/googleidentity/config/TopGuiceServletContextListener.java

```diff
@@ -17,32 +17,32 @@
 package com.google.googleidentity.config;
 
 import com.google.googleidentity.mysql.CloudSqlModule;
-import com.google.googleidentity.oauth2.client.test.TestClientModule;
-import com.google.googleidentity.oauth2.client.test.TestJdbcClientModule;
+import com.google.googleidentity.oauth2.client.seed.InMemoryClientSeedModule;
+import com.google.googleidentity.oauth2.client.seed.JdbcClientSeedModule;
 import com.google.googleidentity.oauth2.config.OAuth2ServerModule;
-import com.google.googleidentity.user.test.TestJdbcUserModule;
-import com.google.googleidentity.user.test.TestUserModule;
+import com.google.googleidentity.user.seed.JdbcUserSeedModule;
+import com.google.googleidentity.user.seed.InMemoryUserSeedModule;
 import com.google.inject.Guice;
 import com.google.inject.Injector;
 import com.google.inject.servlet.GuiceServletContextListener;
 
-/** Start GuiceServlet, Create an Injector for Guice in OAuth2Module{@link OAuth2Module} */
-public final class OAuth2GuiceServletContextListener extends GuiceServletContextListener {
+/** Start GuiceServlet, Create an Injector for Guice in OAuth2Module{@link RequestMappingModule} */
+public final class TopGuiceServletContextListener extends GuiceServletContextListener {
```

Comment:
Since there is no other levels of GuiceServletContextListener, I think we can name it as DemoServletContextListener